Support/Scripts/run-lldb-tests.sh

```diff
@@ -13,8 +13,34 @@
 # against ds2. It requires a few hacks in the testing infra to disable
 # broken unit tests.
 
+cherry_pick_patches() {
+  if ! $opt_fast; then
```

Comment:
You don't need this check, `cherry_pick_patches` is intended to  be called only when you're setting up the repo, so in "non-fast" runs anyway.